external/header.php

```diff
@@ -84,6 +87,11 @@
     return;
 }
 
+if (!extension_loaded('PDO') && Xhgui_Config::read('save.handler') === 'pdo') {
+    error_log('xhgui - extension pdo not loaded');
```

Comment:
the handler should not produce noise on misconfiguration. either remove error log or throw a fatal exception.

Reply:
What about original code above? 
`error_log('xhgui - extension mongo not loaded');`
Should I change that also ?